lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2231,34 +2262,45 @@ shrinkTxBody (Cardano.ShelleyTxBody e bod scripts scriptData aux val) = tail
     shrinkUpdates SNothing = []
     shrinkUpdates (SJust _) = [SNothing]
 
+
+balanceTransaction'
+    :: (UTxOIndex, Wallet (SeqState 'Mainnet ShelleyKey), Set Tx)
+    -> StdGenSeed
+    -> PartialTx
+    -> Either ErrBalanceTx SealedTx
+balanceTransaction' wal seed tx  =
+    flip evalRand (stdGenFromSeed seed) $ runExceptT $ balanceTransaction @(Rand StdGen)
+            (Ctx @(Rand StdGen) nullTracer testTxLayer)
+            (delegationAddress @'Mainnet)
+            mockProtocolParametersForBalancing
+            dummyTimeInterpreter
+            wal
+            tx
```

Comment:
https://input-output-hk.github.io/adrestia/code/Coding-Standards#limit-line-length-to-80-characters

```suggestion
    flip evalRand (stdGenFromSeed seed) $ runExceptT $
        balanceTransaction @(Rand StdGen)
            (Ctx @(Rand StdGen) nullTracer testTxLayer)
            (delegationAddress @'Mainnet)
            mockProtocolParametersForBalancing
            dummyTimeInterpreter
            wal
            tx
```